portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,44 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="bootstrap.min.css">
+  </head>
+
+  <!--Display Nav Bar at top of page-->
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <div class="collapse navbar-collapse" id="navbarColor03">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item">
+          <a class="nav-link" href="index.html">HOME</a>
+        </li>
+        <li class="nav-item active">
+          <a class="nav-link" href="#">ABOUT</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="projects.html">PROJECTS</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="courses.html">COURSES</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
```

Comment:
Not for this PR, but when we get to the server part, we can extract the nav bar html and fetch it for each page so we don't have to enumerate it repeatedly - something like this: https://css-tricks.com/the-simplest-ways-to-handle-html-includes/#article-header-id-6
Consider adding a TODO here as a reminder for that